Mathlib/MeasureTheory/Covering/Besicovitch.lean

```diff
@@ -124,7 +124,9 @@ Finally, the last ball is chosen after all the other ones, meaning that `h` can
 by keeping only one side of the alternative in `hlast`.
 -/
 structure Besicovitch.SatelliteConfig (α : Type*) [MetricSpace α] (N : ℕ) (τ : ℝ) where
+  /-- Centres of the balls -/
```

Comment:
You write "centres" here but "centers" below, choose one!

Reply:
I must have left a rubber duck there. It has been fixed.

Reply:
I think you should use the American spelling to match `Set.center`

Reply:
Please don't forget this.